src/controller/postRouter.js

```diff
@@ -31,18 +31,37 @@ postRouter.get("/", async (req, res) => {
     }
 });
 
-postRouter.patch("/replies", authenticate, validateTextBody, async (req, res) => {
+postRouter.patch("/:id/replies", authenticate, postMiddleware.validateTextBody, async (req, res) => {
     //TODO check song title exists in API
     try {
-        await createReply(res.locals.user.username, req.body.text, req.body.id);
+        await postService.createReply(res.locals.user.itemID, req.body.text, req.params.id);
         res.status(200).json({
-            message: "Reply successfully created"
+            message: `Replied to ${req.params.id} successfully`
         });
     } catch (err) {
         handleServiceError(err, res);
     }
 });
 
+postRouter.patch("/:id/like", authenticate, postMiddleware.validateLike, async (req, res) => {
```

Comment:
Unfortunately, rest does not allow verbs in the url. I can understand why you have it though. The code is so much cleaner when everything has its own route.

Reply:
Maybe likes would work? If not, would thumbsup work or karma (reddit)?

Reply:
OH I SEE. Sorry, I had misunderstood because english is hard. Yeah likes would work and would be better than like. Its crazy how 1 letter changes a word from a verb to a noun